### PR TITLE
Sense monitor documentation

### PR DESCRIPTION
**Pull request by kbickar**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11580

## Checklist:

  - [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.sense.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  email: CLIENT_ID
+  password: CLIENT_SECRET
+```
+
+Two sensors will be created with the following names:
+- **Energy Usage**: Current active power usage in Watts.  Updated every 10 seconds.
+- **Solar Generation**: Current active power generation in Watts.  Updated every 10 seconds.
+- **Daily Energy Usage**: Daily power usage in kWh.  Updated every 2.5 minutes.
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.sense.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Comment by frenck** [comment]
@kbickar Looks 👍 
Can be merged as soon as the parent PR gets merged.

File: source/_components/sensor.sense.markdown [hunk 2]
```diff
@@ -30,11 +30,12 @@ sensor:
     - daily_production
 ```
 
-Two types of sensors can be monitored will be created with the following names:
+Two types of sensors can be monitored and will be created with the following names:
 - **Active Usage/Production**: Current active power usage/production in Watts. Updated every 30 seconds.
 - **Daily Usage/Production**: Daily power usage/production in kWh. Updated every 5 minutes.
+- ...
 
-Weekly, Monthly, and Yearly variants are also available.
```
Comment (PopRe) [inline source/_components/sensor.sense.markdown]: Not a fan of the Oxford comma?

Comment (c727) [reply source/_components/sensor.sense.markdown]: Haha, we don't have this in German^^
If this is your innermost wish you can add it again ;p

**Pull request merged**